src/evm/types.ts

```diff
@@ -0,0 +1,85 @@
+export type TokenBalancesParams = {
```

Comment:
We should double check these params. I think all is right, but something to keep an eye on as we continue to evolve.

Reply:
Yeah we should generate this from OpenAPI